lib/themes/dosomething/paraneue_dosomething/templates/campaign/campaign-gallery.tpl.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ * Returns the HTML for a Campaign Gallery.
+ *
+ * Available Variables
+ * - $items: The Campaign gallery items to display.
+ */
+?>
+<section class="container container--campaigns">
+  <ul class="gallery -mosaic -featured">
```

Comment:
In fenders we trust :smile: 

@sergii-tkachenko @aaronschachter It's part of our front-end approach. So we have classes that define a "module", like the `.gallery` one above, and then we have modifier classes that start with a dash like `.-mosaic` to indicate that it's a specific type of gallery. Additional dashed classes are just extra modifiers on the main module. Hopefully that makes sense. It's a made up system, but it's been pretty great for CSSing :art: